### ISSUE TITLE
thread '<main>' has overflowed its stack

### ISSUE THREAD

**Issue by bvssvni** [PistonDevelopers/glutin_window#48]
When trying to build it I get this error:

```
thread '<main>' has overflowed its stack
Illegal instruction: 4
```

**Comment by bvssvni** [comment]
OSX 10.9.5.

```
rustc 1.3.0-nightly (118a5c4c3 2015-07-21)
cargo 0.4.0-nightly (0faebf2 2015-07-20) (built 2015-07-20)
```

**Comment by bvssvni** [comment]
Seems to be fixed on `rustc 1.3.0-nightly (d33cab1b1 2015-07-22)`. Closing.
